src/runtime/pkg/sev/ovmf.go

```diff
@@ -0,0 +1,101 @@
+// Copyright contributors to AMD SEV/-ES in Go
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package sev
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"os"
+)
+
+// GUID 96b582de-1fb2-45f7-baea-a366c55a082d
+var ovmfTableFooterGuid = guidLE{0xde, 0x82, 0xb5, 0x96, 0xb2, 0x1f, 0xf7, 0x45, 0xba, 0xea, 0xa3, 0x66, 0xc5, 0x5a, 0x08, 0x2d}
```

Comment:
Turn it a constant. Likewise `sevEsResetBlockGuid`.

Reply:
hmmm, I think Go const can only be for boolean, numeric, and string. Here the type is `guidLE` which is defined as an array in sev.go:

```
type guidLE [16]byte
```

What would be the Go syntax for a const array?

Reply:
> What would be the Go syntax for a const array?

I don't think [there is one.](https://blog.boot.dev/golang/golang-constant-maps-slices/). Go--

Reply:
Disregard my suggestion @dubek . Slices are constants in Go already.

Reply:
@wainersm I'm not sure why you say slices are constants. I think bad code can modify `ovmfTableFooterGuid` if it likes (which we don't want). The thing is, I don't know of a Go language mechanism to prevent this; that's why I'm left with this global `var`.

I see that even in the [official uuid package](https://github.com/google/UUID/blob/v1.0.0/hash.go#L15-L18) they keep binary UUIDs as `var`.

Reply:
@dubek when I said "slices are constants" I was thinking in the length, actually the individual elements can be reset, so you are right. Bear in mind I am not golang expert, so one alternative would be to make  `ovmfTableFooterGuid` as a string constant, then you convert to `guidLE` (slice of bytes) when needed.